### PR TITLE
Added R cuts, train configs

### PR DESCRIPTION
**Pull request by hmurakam**

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx [hunk 1]
```diff
@@ -2858,6 +2858,48 @@ Bool_t AliConversionPhotonCuts::SetRCut(Int_t RCut){
...
+    fExcludeMinR = 180.;
+    fExcludeMaxR = 250.;
+    break;
+  case 24: //o
+    fMaxR = 180.;
+    fMinR = 15.;
+    fExcludeMinR = 180.;
+    fExcludeMaxR = 250.;
+    break;
+  case 26: //p
```
Comment (nschmidtALICE) [inline PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx]: you missed case 25 here and thus the letter "p" will not correspond to case 26. Please fix

Comment (hmurakam) [reply PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx]: sorry !

Comment (nschmidtALICE) [reply PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx]: No problem, it would just have crashed for you on the trains then. Thanks for fixing it.

**Comment by nschmidtALICE** [comment]
+1

**Pull request merged**